CompetitionTemplate/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -17,4 +22,27 @@ public void setupMyCommands(
         operatorInterface.leftButtons.getifAvailable(1).whenPressed(myCommand);
     }
     */
+    
+    @Inject
+    public void setupCollectorCommands(
+            OperatorInterface operatorInterface,
+            CollectorIntakeCommand collectorIntakeCommand,
+            CollectorEjectCommand collectorEjectCommand
+            )
+    {
+        operatorInterface.rightButtons.getifAvailable(1).whileHeld(collectorIntakeCommand);
+        operatorInterface.leftButtons.getifAvailable(2).whileHeld(collectorEjectCommand);
+    }
+    
+    @Inject
+    public void setupShiftingCommands(
+            OperatorInterface operatorInterface,
+            ShiftHighCommand shiftHighCommand,
+            ShiftLowCommand shiftLowCommand)
+    {
+        operatorInterface.rightButtons.getifAvailable(3).whileHeld(shiftHighCommand);
```

Comment:
these should just be 'whenPressed' since you should just be able to tap the button to shift, no need to hold it.